metadata/v20/v20/example.json

```diff
@@ -106,11 +106,11 @@
             ],
             "licenses": [
```

Comment:
There, more extensive examples are missing in the license and resource information is missing. In the schema we must add the full example. Currently, we only show a single column. See the example from v16.